src/views/GovernanceReviewTeam/Actions/ProvideGRTFeedbackToBusinessOwner.test.tsx

```diff
@@ -0,0 +1,339 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { MemoryRouter, Route } from 'react-router-dom';
+import { MockedProvider } from '@apollo/client/testing';
+import {
+  render,
+  screen,
+  waitForElementToBeRemoved
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import configureMockStore from 'redux-mock-store';
+
+import { businessCaseInitialData } from 'data/businessCase';
+import { initialSystemIntakeForm } from 'data/systemIntake';
+import AddGRTFeedbackKeepDraftBizCase from 'queries/AddGRTFeedbackKeepDraftBizCase';
+import AddGRTFeedbackProgressToFinal from 'queries/AddGRTFeedbackProgressToFinal';
+import AddGRTFeedbackRequestBizCaseQuery from 'queries/AddGRTFeedbackRequestBizCaseQuery';
+import GetAdminNotesAndActionsQuery from 'queries/GetAdminNotesAndActionsQuery';
+import GetSystemIntakeQuery from 'queries/GetSystemIntakeQuery';
+import Notes from 'views/GovernanceReviewTeam/Notes';
+
+import RequestOverview from '../RequestOverview';
+
+const waitForPageLoad = async () => {
+  await waitForElementToBeRemoved(() => screen.getByTestId('page-loading'));
+};
+
+jest.mock('@okta/okta-react', () => ({
+  useOktaAuth: () => {
+    return {
+      authState: {
+        isAuthenticated: true
+      },
+      oktaAuth: {
+        getAccessToken: () => Promise.resolve('test-access-token'),
+        getUser: () =>
+          Promise.resolve({
+            name: 'John Doe'
+          })
+      }
+    };
+  }
+}));
+
+window.scrollTo = jest.fn();
+
+describe('Provide GRT Feedback to GRT Business Owner', () => {
+  const intakeQuery = {
+    request: {
+      query: GetSystemIntakeQuery,
+      variables: {
+        id: 'a4158ad8-1236-4a55-9ad5-7e15a5d49de2'
+      }
+    },
+    result: {
+      data: {
+        systemIntake: {
```

Comment:
If we're listing out an intake in multiple tests, it might be worth adding a basic intake in a test helper somewhere. We can always spread it in and overwrite the data points relevant to the given test. All of this data setup is making it more difficult to determine what is relevant here and what is just basic setup to make the app work.